crates/interledger-store/src/redis/mod.rs

```diff
@@ -294,8 +294,9 @@ impl RedisStoreBuilder {
         std::thread::spawn(move || {
             #[allow(clippy::cognitive_complexity)]
             let sub_status =
-                sub_connection.psubscribe::<_, _, Vec<String>>(&["*"], move |msg| {
+                sub_connection.psubscribe::<_, _, Vec<String>>(&[format!("{}{}", &db_prefix, "*")], move |msg| {
```

Comment:
wouldn't this subscribe to both, as in the asterisk should be removed?
```suggestion
                sub_connection.psubscribe::<_, _, Vec<String>>(&[format!("{}{}", &db_prefix)], move |msg| {
```

Reply:
I think the `*` means all publish targets https://redis.io/commands/psubscribe though it doesn't really say does it include targets which are created after subscription is started.

Reply:
What do you mean by subscribe to both? I think that pattern needs `*` if we don't match the exact string

Reply:
There is an array of patterns. We have only one. It used to be `*`, and now it is `db_prefix*`, meaning any channel with `db_prefix` prefix. Does it makes sense?

Reply:
Ah, now I can see it, the `"*"` went to the format!. I thought it was going to the psubscribe, now I understand. My suggestion is bad, however now I'd suggest this:

```suggestion
                let prefix = format!("{}*", &db_prefix);
                sub_connection.psubscribe::<_, _, Vec<String>>(&[prefix], move |msg| {
```